test/integration/federation/framework/api.go

```diff
@@ -70,7 +72,9 @@ func (f *FederationAPIFixture) SetUpWithRunOptions(t *testing.T, runOptions *opt
 		t.Fatal(err)
 	}
 
-	f.Host = fmt.Sprintf("http://%s:%d", runOptions.InsecureServing.BindAddress, runOptions.InsecureServing.BindPort)
+	port := strconv.Itoa(runOptions.InsecureServing.BindPort)
+	hostPort := net.JoinHostPort(runOptions.InsecureServing.BindAddress, port)
```

Comment:
Consistency: It seems that the other changes refer to this as `ipPort` - why not here too?